test/e2e/framework/volume/fixtures.go

```diff
@@ -577,7 +588,7 @@ func TestVolumeClient(client clientset.Interface, config TestConfig, fsGroup *in
 // starting and auxiliary pod which writes the file there.
 // The volume must be writable.
 func InjectContent(client clientset.Interface, config TestConfig, fsGroup *int64, fsType string, tests []Test) {
-	injectorPod, err := runVolumeTesterPod(client, config, "injector", fsGroup, tests)
+	injectorPod, err := runVolumeTesterPod(client, config, "injector", true, fsGroup, tests)
```

Comment:
I think we cannot set this to true for all cases. For example, for windows, we cannot run privileged container.

Reply:
Added a check for Windows